File: Aula04/ex02.cpp
// Subtraindo Matrizes//

#include <iostream>
using namespace std;

int main() {
    double A[5], B[5], C[5];

    for(int i = 0; i < 5; i++){
        cout << "A: ";
        cin >> A[i];

        cout << "B: ";
        cin >> B[i];

        C[i] = A[i] - B[i];
    }

    for(int i = 0; i < 5; i++){
        cout << C[i] << " ";
    }

    return 0;
}

File: Aula04/ex02.java
/*Subtraindo Matrizes*/

import java.util.Scanner;

public class Ex2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double[] A = new double[5];
        double[] B = new double[5];
        double[] C = new double[5];

        for(int i = 0; i < 5; i++){
            System.out.print("A: ");
            A[i] = sc.nextDouble();

            System.out.print("B: ");
            B[i] = sc.nextDouble();

            C[i] = A[i] - B[i];
        }

        for(double x : C){
            System.out.print(x + " ");
        }
    }
}